### DISCUSSION TITLE
GitHub app doesn't maintain scroll position in viewed source code file when switching between apps via recent-apps menu in Android, causing the app to scroll to top of the file upon return

### DISCUSSION THREAD

**Discussion by D-Pow** [community/community#55679] [Mobile]
While viewing a file in the source code in the mobile app (Android 13, Samsung Galaxy S22, One UI version 5.1), I will scroll down to view lines of code later in the file. Then, I'll use Android's recent-apps button/swipe gesture to switch to a different app. Upon returning to the GitHub app, the file I was viewing before doesn't maintain the scroll position I was at before; instead, it resets the scroll position to the top of the file, making me have to manually scroll all the way back down to find the line of code I was viewing previously.

Could this be fixed to maintain scroll position when switching between apps, please? It would make usage of the app more efficient and user-friendly. Thanks!

**Comment by aconsuegra** [comment]
Hi @D-Pow, thanks for reaching out with your feedback! I have been able to reproduce this issue, and it looks to me like a bug, so I will create an internal issue for our engineers. I hope we can work on a fix soon. Thanks again!

**Reply by D-Pow** [reply]
Cool, thanks for the prompt reply! I'm a dev too, and I wouldn't mind helping out where I can, but I realize the mobile app source code isn't open source so I understand if you'd rather I not. Either way, I appreciate your attention to this matter :)
